content/topics/node-and-sql/_index.md

```diff
@@ -2,4 +2,24 @@
 _db_id: 44
 content_type: topic
 title: Introduction to Node and SQL
----
\ No newline at end of file
+---
+
+#### In this topic, you will learn how to connect to a PostgreSQL database using node
+
+- node-Postgres is a collection of Node.js modules used to interact with a PostgreSQL database.
+- node-Postgres has many features like callbacks, connection pooling, prepared statements, async/await, and promises.
+- connection pooling allows your application to reuse existing database connections by automatically saving the connection to a pool so it can be reused instead of having to create a new connection every time data is being retrieved/accessed. An example of this can be a cache component where data is frequently accessed and stored.
+
+### Connection pooling
+
+#### Client
+
+- [Node & SQL tutorial using Client](https://node-postgres.com/api/client)
```

Comment:
This is not a tutorial.  Tutorial is not a word that describes the api docs linked to here.